apps/mark/backend/tsconfig.build.json

```diff
@@ -10,10 +10,8 @@
     "declarationMap": true
   },
   "references": [
-    { "path": "../../../libs/api/tsconfig.build.json" },
+    { "path": "../../../libs/auth/tsconfig.build.json" },
     { "path": "../../../libs/basics/tsconfig.build.json" },
-    { "path": "../../../libs/backend/tsconfig.build.json" },
-    { "path": "../../../libs/db/tsconfig.build.json" },
```

Comment:
Looks like a few libs were specified here that aren't actually used, a la copy paste

Reply:
nice catch!